src/models/slices/filtersearch.ts

```diff
@@ -0,0 +1,10 @@
+import { FilterSearchResponse } from '@yext/answers-core';
+
+interface FilterSearchState {
+  query: string,
+  results?: FilterSearchResponse
```

Comment:
Since we are removing the response models from answers-headless, can we instead spread the relevant properties into the FilterSearchState? We'll want sectioned, sections, results, and inputIntents (in headless we've been calling them searchIntents, so I think we should go with that for consistency). Also, the queryId should update state.query.queryId and the UUID should update state.meta.uuid (I'm still working on that change for regular searches and for autocomplete)

Reply:
> Since we are removing the response models from answers-headless, can we instead spread the relevant properties into the FilterSearchState? We'll want sectioned, sections, results, and inputIntents (in headless we've been calling them searchIntents, so I think we should go with that for consistency). 
oh I see, will do

> Also, the queryId should update state.query.queryId and the UUID should update state.meta.uuid (I'm still working on that change for regular searches and for autocomplete)
Not sure if I follow this right, but the query from filterSearch is different from the other queries store in QueryState so would storing queryId in there make sense? similar thing with uuid.

Reply:
That's a good point that it doesn't really make sense to put it in the top level query since this has it's own different query. I'd keep queryId and uuid in the FilterSearchState for now then. I'm going to discuss this with Tom